src/main/java/com/home/eshop/dao/BikesDao.java

```diff
@@ -6,15 +6,15 @@
 import java.util.ArrayList;
 import java.util.List;
 
-public class BikesDao {
-    private ArrayList<Bike> bikes = new ArrayList<Bike>();
+public class BikesDao implements DaoInterface {
     private File file;
     static int maxIdBike = 0;
 
     public BikesDao(String path) {
         this.file = new File(path);
     }
-    private Bike addId(Bike bike) {
+
+    Bike addId(Bike bike) {
```

Comment:
rename to be 'add'

Reply:
or save

Reply:
Rename to idProcessing.
Mission Complete